cirq/study/sweepable.py

```diff
@@ -78,6 +78,26 @@ def to_sweep(sweep_or_resolver_list: Union['Sweep', ParamResolverOrSimilarType,
         'Unexpected sweep-like value: {}'.format(sweep_or_resolver_list))
 
 
+def dict_list_to_sweep(dict_list: Iterable[Dict[str, float]]):
+    """Converts a list of dicts into a Zipped sweep
+
+    Args:
+        dict_list: this must be a sequence of dicts.  Each dict must have the
```

Comment:
nit: sequence -> iterable

In python sequences have`__len__` and `__getitem__` while iterables have `__iter__`.  Your type is correct as you only use iterating over the list.

Reply:
Removed this function.